cartographer_ros/cartographer_ros/assets_writer.cc

```diff
@@ -178,7 +180,7 @@ void AssetsWriter::Run(const std::string& configuration_directory,
           lua_parameter_dictionary->GetDictionary("pipeline").get());
   const std::string tracking_frame =
       lua_parameter_dictionary->GetString("tracking_frame");
-
+  rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_SYSTEM_TIME);
```

Comment:
Hm, it feels like we want this to be `RCL_ROS_TIME`, but I'm not 100% sure what this utility is doing.

Reply:
Done